install.sh

```diff
@@ -105,13 +105,15 @@ if lspci | grep -E -i '(vga|3d|display)' | grep -i nvidia > /dev/null; then
 # AMD
 elif lspci | grep -E -i '(vga|3d|display)' | grep -i amd > /dev/null; then
 	if lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon, amdgpu' > /dev/null; then
-		echo "AMD GPU with radeon kernel detected, switching kernel module and installing drivers..."
+		echo "AMD graphics card with radeon kernel module detected, switching kernel module and installing drivers..."
 		echo "blacklist radeon" >> /etc/modprobe.d/gameros.conf
 		echo "options amdgpu si_support=1" >> /etc/modprobe.d/gameros.conf
 		echo "options amdgpu cik_support=1" >> /etc/modprobe.d/gameros.conf
 		echo "options radeon si_support=0" >> /etc/modprobe.d/gameros.conf
 		echo "options radeon cik_support=0" >> /etc/modprobe.d/gameros.conf
 		pacman --noconfirm -S \
+			mesa \
+			lib32-mesa \
```

Comment:
this is unnecessary, these packages are already installed with the base set of packages

Reply:
No they it isnt, if it is then it will include 64-bit mesa and ignore 32 bit mesa one which is a must have for proton

Reply:
I ran the installer without,the gpu section and both 64 and 32 bit mesa libraries were installed